src/Syntax_Client/Oni_Syntax_Client.rei

```diff
@@ -41,4 +42,5 @@ let healthCheck: t => unit;
 let close: t => unit;
 
 // Debug methods
-let simulateException: t => unit;
+let simulateReadException: t => unit;
+let simulateMessageException: t => unit;
```

Comment:
I would put these in a `Testing` module. Descriptive code is always better than comments, since comments are easily ignored and quickly get outdated and become misinforming. a sub-module also prevents them from showing up among "normal" completions, where comments don't help at all.

Reply:
Good idea! 👍